code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -61,7 +61,7 @@ \subsection{Table of Units}
 \end{longtable}
 \subsection{Table of Symbols}
 \label{Sec:ToS}
-The symbols used in this document are summarized in \hyperref[Table:ToS]{Tab: ToS} along with their units. Throughout the document, values with a subscript $i$ implies that the value will be taken at and analyzed at a slice or slice interface composing the total slip mass. For vector quantities, the units shown are for each component of the vector.
+The symbols used in this document are summarized in \hyperref[Table:ToS]{Tab: ToS} along with their units. Throughout the document, a subscript $i$ indicates that the value will be taken at and analyzed at a slice or slice interface composing the total slip mass. For vector quantities, the units shown are for each component of the vector.
```

Comment:
I like the new sentence, but reading it again, I think some commas would make it bit better.  :smile:  Could we change it to "... the value will be taken at, and analyzed at, ...

Reply:
Done in 919608d.